android/app/src/main/res/values/strings.xml

```diff
@@ -182,10 +190,10 @@ Contributor by reason of your accepting any such warranty or additional
 liability.</p> ]]></string>
     <string name="privacy_policy_text"><![CDATA[To proceed please accept the <a href="https://www.jetbrains.com/company/privacy.html">Privacy Policy</a>]]></string>
     <string name="privacy_policy_accept">I accept the Privacy Policy</string>
-    <string name="voting_code_prompt">If you would like to vote for a session enter your voting code</string>
+    <string name="rating_code_prompt">If you would like to rate a session enter your code</string>
     <string name="submit_button">Submit</string>
-    <string name="code_verification_failed">Failed to verify code</string>
+    <string name="code_verification_failed">Failed to verify the code</string>
     <string name="code_incorrect">Sorry, the code entered is incorrect</string>
-    <string name="voting_text">In order to enable voting, please enter your voting code</string>
-    <string name="verify_button_text">Verify voting code</string>
+    <string name="rating_text">In order to enable rating, please enter your code</string>
+    <string name="verify_button_text">Verify rating code</string>
```

Comment:
I'd drop "rating" and just use code.

Reply:
addressed in https://github.com/JetBrains/kotlinconf-app/pull/39